File: Frontend/src/interfaces/v12/stockIn/stockInDetails/stockInItemRemarkDetailsDto.ts
import { guid } from "types/guid";

export interface StockInItemRemarkDetailsDto {
  id: guid;
  stockInItemId: guid;
  productRemarkId: guid;
  remark: string;
  createdAt?: string;
  createdById?: guid;
  changedAt?: string;
  changedById?: guid;
}

File: Wms.Api/Dto/StockIn/StockInDetails/StockInItemRemarkDetailsDto.cs
namespace Wms.Api.Dto.StockIn.StockInDetails
{
    public class StockInItemRemarkDetailsDto
    {
        public Guid Id { get; set; }
        public Guid StockInItemId { get; set; }
        public Guid ProductRemarkId { get; set; }
        public string Remark { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public Guid CreatedById { get; set; }
        public DateTime? ChangedAt { get; set; }
        public Guid? ChangedById { get; set; }
    }
}
